contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -5325,3 +5336,169 @@ pltsql_remove_current_query_env(void)
 		destroy_failed_transactions_map();
 	} 
 }
+
+/*
+ * Drop statement of babelfish, currently delete extended property as well.
+ */
+static void
+bbf_ExecDropStmt(DropStmt *stmt)
+{
+	int16			db_id;
+	char			*physical_schema_name = NULL,
+					*schema_name = NULL,
+					*major_name = NULL;
+	const char		*logical_schema_name = NULL;
+	ObjectAddress	address;
+	Relation		relation = NULL;
+	Oid				schema_oid;
+	ListCell		*cell;
+
+	db_id = get_cur_db_id();
+
+	if (stmt->removeType == OBJECT_SCHEMA)
+	{
+		foreach(cell, stmt->objects)
+		{
+			/* Get schema_name */
+			logical_schema_name = get_logical_schema_name(strVal(lfirst(cell)),
+														  true);
+
+			if (logical_schema_name)
+			{
+				schema_name = downcase_identifier(logical_schema_name,
```

Comment:
why do we do downcase for identifier? 
Shouldn't the original case be stored in the extended property catalog table?

Reply:
In PG, all name is in downcase and we should store downcase in the extended property catalog table.

sp_xxxextendedproperty can pass different case. For example,
```
-- we have schema dbo
sp_addextendedproperty 'name', 'value', 'schema', 'Dbo'
```